Makefile

```diff
@@ -64,10 +65,14 @@ binary: clean all
 
 deps:
 	@REBAR_EXTRA_DEPS=1 ./rebar get-deps
-	@(cd deps/retest && $(REBAR) compile escriptize)
+	$(MAKE) -C deps/retest debug
```

Comment:
I thought so. I have just put the debug there, so it's equvivalent with the previous behaviour (see rebar.config in deps/retest). I will remove it in a fixup.